File: mcs-java/src/main/java/anc/models/GeneticConfiguration.java
package anc.models;

public class GeneticConfiguration {
    public final int nGenomes;
    public final int nGenerations;
    public final int nCb;
    public final int nTribes;
    public final double pCrossover;
    public final double pMutation;


    public GeneticConfiguration(int nGenomes, int nGenerations, int nCb, int nTribes, double pCrossover,
            double pMutation) {
        this.nGenomes = nGenomes;
        this.nGenerations = nGenerations;
        this.nCb = nCb;
        this.nTribes = nTribes;
        this.pCrossover = pCrossover;
        this.pMutation = pMutation;
    }

    public static GeneticConfiguration fromPaper() {
        return new GeneticConfiguration(256, 512, 32, 4, 0.9, 0.5);
    }
}

File: Mcs-cs/Mcs/Models/GeneticConfiguration.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Mcs.Models
{
    public class GeneticConfiguration
    {
        public int NGenomes;
        public int NGenerations;
        public int NCb;
        public int NTribes;
        public double PCrossover;
        public double PMutation;

        public GeneticConfiguration(int nGenomes, int nGenerations, int nCb, int nTribes, double pCrossover,
            double pMutation)
        {
            NGenomes = nGenomes;
            NGenerations = nGenerations;
            NCb = nCb;
            NTribes = nTribes;
            PCrossover = pCrossover;
            PMutation = pMutation;
        }

        public static GeneticConfiguration FromPaper()
        {
            return new GeneticConfiguration(256, 512, 32, 4, 0.9, 0.5);
        }

    }
}
